andela/codango
# Create Post feed on home page

#### What does this PR do?
Create Post feed on home page with a search bar in a submenu
#### Description of Task to be completed?
 - Create feeds section in the home component
 - Create Popular section
 - Create a search bar

#### How should this be manually tested?
- Visit the review app [here](https://codango-staging-pr-162.herokuapp.com)
- Click on the "new" navigation menu to see the home page

#### What are the relevant pivotal tracker stories?
128843949 - Create the home page listing section with search functionality.

#### Screenshots
![screen shot 2016-09-28 at 2 16 31 pm](https://cloud.githubusercontent.com/assets/17270426/18914725/39f3e198-8586-11e6-858b-5a3faae60bd8.png)

Feeds flow under the Submenu
![screen shot 2016-10-10 at 11 28 05 am](https://cloud.githubusercontent.com/assets/17270426/19233324/b72fa8cc-8edc-11e6-85ef-3bc966f92d3c.png)
